src/enfa.ts

```diff
@@ -97,31 +118,47 @@ class EpsilonNFABuilder {
         }
 
         return {
+          type: 'BuildChildResult',
           initialState: q0,
           acceptingState: f0,
         };
       }
       case 'Sequence': {
         // ^または$が有効的な場所にない場合、エラーを返す
-        node.children.forEach((child, index) => {
-          if (index > 0 && child.type === 'LineBegin') {
-            throw new Error('Illigal use LineBegin');
+        for (let i = 0; i < node.children.length; i++) {
+          if (i > 0 && node.children[i].type === 'LineBegin') {
+            return { type: 'Error', error: new Error('Illigal use LineBegin') };
```

Comment:
このコミットのものではないですが、以下のほうが良い気がしたのでついでに修正してもらいたいです……。

```suggestion
            return { type: 'Error', error: new Error('Illigal use of LineBegin') };
```